import-export-cli/integration/testutils/app_testUtils.go

```diff
@@ -82,6 +83,7 @@ func importAppPreserveOwner(t *testing.T, sourceEnv string, app *apim.Applicatio
 
 	t.Cleanup(func() {
 		client.DeleteApplicationByName(app.Name)
+		base.WaitForIndexing()
```

Comment:
No need to wait for indexing here as applications are not solr indexed on APIM side.